c99sh

```diff
@@ -41,18 +43,20 @@ If compilation is successful, the exit status is that of PROGRAM.
 "
 declare -i e=0 m=0 s=0 v=0 l=0
 declare -A E=() L=()
-while getopts e:hl:mp:r:st:vx: flag; do
+while getopts e:hl:mp:r:st:vx:F:L: flag; do
   case $flag in
-    e) E[$e]=$OPTARG; ((e+=1))    ;; # To be processed later; notice 'E' not 'e'
+    e) E[$e]=$OPTARG; ((e+=1))    ;; # To process later; notice 'E' not 'e'
     h) exec echo -ne "$help"      ;; # Stops processing due to exec
     l) L[$l]=$OPTARG; ((l+=1))    ;; # Libraries to be processed later
-    m) m=1                        ;; # To be processed later
+    m) m=1                        ;; # To process later
     p) pkg[$OPTARG]="flag -$flag" ;; # Additively record requested package
     r) r=$OPTARG                  ;; # Record requested rcfile, last one wins
-    s) s=1                        ;; # To be processed later
-    t) T=$OPTARG;                 ;; # To be processed later; notice 'T' not 't'
+    s) s=1                        ;; # To process later
+    t) T=$OPTARG;                 ;; # To process later; notice 'T' not 't'
     v) ((v+=1))                   ;; # Try -v, -vv, etc. to increase verbosity
-    x) X=$OPTARG                  ;; # To be processed later; notice 'X' not 'x'
+    x) X=$OPTARG                  ;; # To process later; notice 'X' not 'x'
+    F) CFLAGS="${CFLAGS:+${CFLAGS} }-$OPTARG"    ;; # Space-separated append
+    L) LDFLAGS="${LDFLAGS:+${LDFLAGS} }-$OPTARG" ;; # Space-separated append
```

Comment:
These are the bashisms to use existing variables or to append.  Again, note the extra `-`.

Reply:
The new functionality is added solely to this block and the help message above, which is nice.